MediaBrowser.Model/Net/MimeTypes.cs

```diff
@@ -98,6 +100,12 @@ public static class MimeTypes
         private static readonly Dictionary<string, string> _extensionLookup = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
         {
             // Type application
+            { "application/vnd.comicbook-rar", ".cbr" },
+            { "application/vnd.comicbook+zip", ".cbz" },
+            { "application/x-cb7", ".cb7" },
+            { "application/x-cba", ".cba" },
+            { "application/x-cbr", ".cbr" },
+            { "application/x-cbt", ".cbt" },
```

Comment:
Why not add all of these to `_mimeTypeLookup`?

Reply:
Good catch. I only did this, once I noticed the mismatch between `application/x-cbr` and the returned filtype `cbz`. I'll add the missing ones. :slightly_smiling_face:

Reply:
Done.